{{ cookiecutter.__project_name_kebab }}/tox.ini

```diff
@@ -3,10 +3,8 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.7,3.8,3.9}-django{3.2}-wagtail{2.15,4.0,4.1}-{sqlite,postgres}
-    python{3.10}-django{3.2}-wagtail{2.15,4.0,4.1,main}-{sqlite,postgres}
-    python{3.7,3.8,3.9,3.10}-django{4.0}-wagtail{4.0,4.1,main}-{sqlite,postgres}
-    python{3.8,3.9,3.10,3.11}-django{4.1,main}-wagtail{4.1,main}-{sqlite,postgres}
+    python{3.7,3.8,3.9,3.10}-django{3.2}-wagtail{4.1,4.2,5.0,main}-{sqlite,postgres}
+    python{3.8,3.9,3.10,3.11}-django{4.1,4.2,main}-wagtail{4.1,4.2,5.0,main}-{sqlite,postgres}
```

Comment:
The CI matrixes (postgres and sqlite) in https://github.com/wagtail/cookiecutter-wagtail-package/blob/main/%7B%7B%20cookiecutter.__project_name_kebab%20%7D%7D/.github/workflows/test.yml need to be updated in line with the changes to the tox envs.